### ISSUE TITLE
quarkus-integration-test-resteasy-jackson-999-SNAPSHOT-runner fails on Windows with latest JDK 17-ea

### ISSUE THREAD

**Issue by jerboaa** [graalvm/mandrel#419]
The native build fails with:

```
========================================================================================================================
GraalVM Native Image: Generating 'quarkus-integration-test-resteasy-jackson-999-SNAPSHOT-runner' (executable)...
========================================================================================================================
[1/7] Initializing...                                                                                   (24.9s @ 0.12GB)
 Version info: 'GraalVM 22.3.0-dev86e8eba0 Java 17 Mandrel Distribution'
 Java version info: '17.0.5-beta+4-202208302337'
 C compiler: cl.exe (microsoft, x64, 19.29.30146)
 Garbage collector: Serial GC
 4 user-specific feature(s)
 - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
 - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.jboss.threads] categories
 - io.quarkus.runtime.graal.ResourcesFeature: Register each line in META-INF/quarkus-native-resources.txt as a resource on Substrate VM
 - org.graalvm.home.HomeFinderFeature: Finds GraalVM paths and its version number
[2/7] Performing analysis...  [*]                                                                       (94.2s @ 1.58GB)
  10,832 (89.91%) of 12,048 classes reachable
  15,018 (56.91%) of 26,389 fields reachable
  52,039 (63.98%) of 81,333 methods reachable
     374 classes,   110 fields, and 1,206 methods registered for reflection
       4 native libraries: crypt32, ncrypt, psapi, winhttp
Fatal error: org.graalvm.compiler.debug.GraalError: com.oracle.svm.util.ReflectionUtil$ReflectionUtilError: java.lang.NoSuchMethodException: sun.security.mscapi.CKeyStore.loadKeysOrCertificateChains(java.lang.String)

	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisFuture.setException(AnalysisFuture.java:49)
------------------------------------------------------------------------------------------------------------------------
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:269)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisFuture.ensureDone(AnalysisFuture.java:63)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisElement.lambda$execute$2(AnalysisElement.java:170)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool.externalHelpQuiescePool(ForkJoinPool.java:2104)
	at java.base/java.util.concurrent.ForkJoinPool.awaitQuiescence(ForkJoinPool.java:3321)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.complete(CompletionExecutor.java:243)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis.doTypeflow(PointsToAnalysis.java:541)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis.finish(PointsToAnalysis.java:529)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.AbstractAnalysisEngine.runAnalysis(AbstractAnalysisEngine.java:143)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.runPointsToAnalysis(NativeImageGenerator.java:746)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.doRun(NativeImageGenerator.java:579)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGenerator.run(NativeImageGenerator.java:536)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.buildImage(NativeImageGeneratorRunner.java:403)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.build(NativeImageGeneratorRunner.java:580)
                        6.5s (5.4% of total time) in 21 GCs | Peak RSS: 2.62GB | CPU load: 1.77
========================================================================================================================
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner.main(NativeImageGeneratorRunner.java:128)
Failed generating 'quarkus-integration-test-resteasy-jackson-999-SNAPSHOT-runner' after 1m 59s.
Caused by: com.oracle.svm.util.ReflectionUtil$ReflectionUtilError: java.lang.NoSuchMethodException: sun.security.mscapi.CKeyStore.loadKeysOrCertificateChains(java.lang.String)
	at org.graalvm.nativeimage.base/com.oracle.svm.util.ReflectionUtil.lookupMethod(ReflectionUtil.java:82)
	at org.graalvm.nativeimage.base/com.oracle.svm.util.ReflectionUtil.lookupMethod(ReflectionUtil.java:69)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.JNIRegistrationUtil.method(JNIRegistrationUtil.java:91)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SecurityServicesFeature.registerSunMSCAPIConfig(SecurityServicesFeature.java:466)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisElement$ElementNotification.lambda$notifyCallback$0(AnalysisElement.java:102)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	... 18 more
Caused by: java.lang.NoSuchMethodException: sun.security.mscapi.CKeyStore.loadKeysOrCertificateChains(java.lang.String)
	at java.base/java.lang.Class.getDeclaredMethod(Class.java:2675)
	at org.graalvm.nativeimage.base/com.oracle.svm.util.ReflectionUtil.lookupMethod(ReflectionUtil.java:74)
	... 23 more
Error: Image build request failed with exit status 1
```

I'm pretty sure this is an upstream graal issue and surfaces only with a JDK >= 17.0.5+4 which includes https://bugs.openjdk.org/browse/JDK-6782021 (see commit: https://github.com/openjdk/jdk17u-dev/commit/e3c178d6d87c9e0e2a75cddc72d802cf9124420e).

**Comment by jerboaa** [comment]
@Karm Do you mind taking a look?

**Comment by jerboaa** [comment]
Seems fixed upstream with 0c88c7d5ded86bf566ba04660f6588c794131add
